doc/man7/EVP_RAND-SEED-SRC.pod

```diff
@@ -0,0 +1,87 @@
+=pod
+
+=head1 NAME
+
+EVP_RAND-SEED-SRC - The randomness seed source EVP_RAND implementation
+
+=head1 DESCRIPTION
+
+Support for deterministic random number generator seeding through the
+B<EVP_RAND> API.
+
+The seed sources used are specified at configuration time using the
```

Comment:
"configuration time" is a little ambiguous here. For users of OpenSSL they would probably interpret this as the time that OpenSSL reads its config file. Perhaps change this to "The seed sources used are specified at the time OpenSSL is built using..."

Reply:
Okay.